include/deal.II/fe/fe_q_base.h

```diff
@@ -324,6 +324,12 @@ class FE_Q_Base : public FE_Poly<POLY,dim,spacedim>
    * Mutex for protecting initialization of restriction and embedding matrix.
    */
   mutable Threads::Mutex mutex;
+
+  /*
+   * For the FE_Q_Bubbles class the highest polynomial degree is
+   * greater than the degree of FE_Q by one.
+   */
```

Comment:
q_degree should describe the degree of th eunderlying polyynomials space. Thus, it is not greater but equal to q_degree for FE_Q(p). The reason for distinguish between q_degree and degree is exactly that one. While for FE_Q(p) these are the same for FEQ_Bubbles(p) the underlying TensorProductPolynomials space is the same but the bubble enrichment polynomals have degree p+1. This way we use the correct quadrature rule.